src/main/java/kostka/moviecatalog/service/ExternalCommentService.java

```diff
@@ -19,7 +19,7 @@
  */
 @Service
 public class ExternalCommentService {
-    public static final long DEFAULT_ID = 999L;
+    public static final long DEFAULT_ID = 201L;
```

Comment:
K čemu je tam to ID vlastně potřeba?

Popravdě, nevím, jestli je dobrý přístup vracet ID nějakého existujícího usera. Co když ten se taky časem smaže? Nebo co když bude nedostupný díky právům atd?

Reply:
pravda, vratim radsi empty list :)